src/main/java/org/opensearch/ad/rest/RestAnomalyDetectorJobAction.java

```diff
@@ -120,4 +216,5 @@ public List<ReplacedRoute> replacedRoutes() {
                 )
             );
     }
+    */
```

Comment:
Do we need this anymore since we have added routes already?

Reply:
I just kept it for reference for the stop detector workflow. But I can remove it and just add the stop detector route handler under `routeHandlers` and comment it out for now.